input/pagecontent/changes.md

```diff
@@ -177,9 +177,13 @@ This change log documents the significant updates and resolutions implemented fr
   - corrected the Observation combined search parameter 'patient+category+status' from SHALL to SHOULD [FHIR-45390](https://jira.hl7.org/browse/FHIR-45390)
   - corrected narrative of Location identifier parameter requester requirements on providing both the system and code values from SHOULD to MAY [FHIR-46209](https://jira.hl7.org/browse/FHIR-46209)
   - replaced security and authorization requirements with pointer to Security and Privacy page [FHIR-46067](https://jira.hl7.org/browse/FHIR-46067)
+  - corrected structure of Patient identifier search parameter responder requirements to support both system and code values [FHIR-47154](https://jira.hl7.org/browse/FHIR-47154)
```

Comment:
This change entry doesn't work. The conformance verb was corrected to SHALL not SHALL NOT in the documentation. You can say the documentation for the Patient identifier search parameter was corrected from "The responder SHALL NOT support only code values." to "The responder SHALL support both."

Reply:
Noted. Removed both entries.